SpatialGDK/Source/SpatialGDK/Private/SpatialGDKSettings.cpp

```diff
@@ -64,6 +73,42 @@ void USpatialGDKSettings::PostEditChangeProperty(FPropertyChangedEvent& Property
 			FModuleManager::LoadModuleChecked<ISettingsModule>("Settings").ShowViewer("Project", "SpatialGDKEditor", "Editor Settings");
 		}
 	}
+
+	if (PropertyName == GET_MEMBER_NAME_CHECKED(USpatialGDKSettings, ActorGroups))
```

Comment:
This makes more sense as an else-if ladder or refactored into helper functions for each property with this function being the dispatcher.

Reply:
More specifically what I'm asking for is a helper which checks which validation functions should be called based on the property name - `RequiresQBIMessageDialog(PropertyChangedEvent)`, `RequiresOffloadingSettingsValidation(PropertyChangedEvent)` etc.